recipes/pvrtc-compressor/all/CMakeLists.txt

```
@@ -0,0 +1,42 @@
+cmake_minimum_required(VERSION 3.1)
```

```suggestion
cmake_minimum_required(VERSION 3.4)
```

CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS requires 3.4